app/src/main/java/org/schabi/newpipe/download/DeleteDownloadManager.java

```diff
@@ -0,0 +1,158 @@
+package org.schabi.newpipe.download;
+
+import android.app.Activity;
+import android.os.Bundle;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.design.widget.BaseTransientBottomBar;
+import android.support.design.widget.Snackbar;
+import android.view.View;
+
+import org.schabi.newpipe.R;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.concurrent.TimeUnit;
+
+import io.reactivex.Completable;
+import io.reactivex.Observable;
+import io.reactivex.android.schedulers.AndroidSchedulers;
+import io.reactivex.disposables.Disposable;
+import io.reactivex.schedulers.Schedulers;
+import io.reactivex.subjects.PublishSubject;
+import us.shandian.giga.get.DownloadManager;
+import us.shandian.giga.get.DownloadMission;
+
+public class DeleteDownloadManager {
```

Comment:
Maybe this should be called DeleteDownloadsManager. In order to reduce confusion.